simapp/upgrades.go

```diff
@@ -48,9 +46,7 @@ func (app SimApp) RegisterUpgradeHandlers() {
 				protocolpooltypes.StoreKey,
 				epochstypes.StoreKey,
 			},
-			Deleted: []string{
-				crisistypes.StoreKey, // The SDK discontinued the crisis module in v0.51.0
-			},
+			Deleted: []string{"crisis"},
```

Comment:
can we keep the comment and mention this in x/upgrading.md?

Reply:
done